### DISCUSSION TITLE
How can I import XML file and insert into Oracle Database?

### DISCUSSION THREAD

**Discussion by ashokeir** [dbeaver/dbeaver#19937] [Q&A]
### Description

Need to see the way to import XML and insert into Oracle database table. I'm using DBeaver Universal Database Manager.

### DBeaver Version

Universal Database Manager

### Operating System

Windows 10 - 64 bit

### Database and driver

Oracle database "anpsebtstscidb.pacsuncorp.com"

### Steps to reproduce

No idea need steps

**Comment by E1izabeth** [comment]
Please take a look at this guide https://dbeaver.com/2022/06/23/import-data-with-dbeaver/

**Comment by ashokeir** [comment]
> Please take a look at this guide https://dbeaver.com/2022/06/23/import-data-with-dbeaver/

I have a standard version not pro one

**Comment by E1izabeth** [comment]
It's only DBeaver PRO feature.

**Comment by bubnenkoff** [comment]
You could use external tool like SmartXML. Please check this example: https://redata.dev/smartxml/docs/creating-database-table-patches-based-on-xml.html
